src/dbnode/network/server/tchannelthrift/node/service.go

```diff
@@ -680,12 +680,16 @@ func (s *service) fetchTagged(ctx context.Context, db storage.Database, req *rpc
 		encodedDataResults = make([][][]xio.BlockReader, results.Size())
 	}
 	if err := s.fetchReadEncoded(ctx, db, response, results, nsID, nsIDBytes, callStart, opts, fetchData, encodedDataResults); err != nil {
+		s.metrics.fetchTagged.ReportError(s.nowFn().Sub(callStart))
 		return nil, err
 	}
 
-	// Step 2: If fetching data read the results of the asynchronuous block readers.
+	// Step 2: If fetching data read the results of the asynchronous block readers.
 	if fetchData {
-		s.fetchReadResults(ctx, response, nsID, encodedDataResults)
+		if err := s.fetchReadResults(ctx, response, nsID, encodedDataResults); err != nil {
```

Comment:
Nice catch... almost tempted to turn on the linter that forces error checking, but that will probably generate huge amounts of noise

Reply:
Yeah, in an ideal world we'd have that on I think

Reply:
It's fine to do "_ = foo" to get around the linter if you run into that @arnikola for cases where you do really want to ignore the error.